ambry-cloud/src/main/java/com/github/ambry/cloud/azure/ADAuthBasedStorageClient.java

```diff
@@ -100,6 +102,18 @@ protected BlobServiceAsyncClient buildBlobServiceAsyncClient(HttpClient httpClie
 
   @Override
   protected void validateABSAuthConfigs(AzureCloudConfig azureCloudConfig) {
+    if (storageAccountInfo() != null) {
+      if (storageAccountInfo().getStorageScope().isEmpty()) {
```

Comment:
Here and at all other places with similar checks, should we check for both null and isEmpty()
Maybe we can define a method in Utils.java like `isNullOrEmpty()`

Reply:
Done